### ISSUE TITLE
[Bug] AppFlowy GitHub home image problem in Mobile View

### ISSUE THREAD

**Issue by wharjack** [AppFlowy-IO/AppFlowy#729]
![Screenshot_20220727-204725](https://user-images.githubusercontent.com/52787179/181251177-9f1747eb-e6bb-4fc7-85c8-e467b1b87a7e.jpg)

i viewed other repository with picture in their home view using my phone and it's okay but appflowy isn't.

**Comment by apomalyn** [comment]
Here is the [line](https://github.com/AppFlowy-IO/AppFlowy/blob/main/README.md?plain=1#L41) that cause the issue:

```
<p align="center"><img src="https://github.com/AppFlowy-IO/appflowy/blob/main/doc/imgs/howtostar.gif" alt="AppFlowy Github" width="1000px" /></p>
```

Removing the `width="1000px"` should fix it.

**Comment by wharjack** [comment]
![image](https://user-images.githubusercontent.com/52787179/181391435-990b29ac-bf6e-446c-af9a-379a04bc13a1.png)

Desktop view

**Comment by wharjack** [comment]
width: 100%; would be good?

**Comment by annieappflowy** [comment]
@wharjack thanks for bringing this up. 100% should work. There is a PR for this fix: https://github.com/AppFlowy-IO/AppFlowy/pull/663. We're waiting for the contributor to format the commit message in order to get the PR merged.

**Comment by MikeWallaceDev** [comment]
I fixed the comments for PR #663, I will merge it as soon as CI finishes.
